data2bids.m

```diff
@@ -1342,16 +1342,21 @@
 
 %% need_motion_json
 if need_motion_json
-  motion_json.POSChannelCount                = sum(strcmpi(hdr.chantype, 'POS'));
-  motion_json.ORNTChannelCount               = sum(strcmpi(hdr.chantype, 'ORNT'));
-  motion_json.VELChannelCount                = sum(strcmpi(hdr.chantype, 'VEL'));
-  motion_json.ANGVELChannelCount             = sum(strcmpi(hdr.chantype, 'ANGVEL'));
-  motion_json.ACCChannelCount                = sum(strcmpi(hdr.chantype, 'ACC'));
-  motion_json.ANGACCChannelCount             = sum(strcmpi(hdr.chantype, 'ANGACC'));
-  motion_json.MAGNChannelCount               = sum(strcmpi(hdr.chantype, 'MAGN'));
-  motion_json.JNTANGChannelCount             = sum(strcmpi(hdr.chantype, 'JNTANG'));
+  motion_json.SamplingFrequency     = hdr.Fs;
+  motion_json.StartTime             = nan;
+  motion_json.MotionChannelCount    = hdr.nChans;
```

Comment:
This is specified in a motion-specific field in the motion.json and not per tracking system. (See google doc https://docs.google.com/document/d/1iaaLKgWjK5pcISD1MVxHKexB3PZWfE2aAC5HF_pCZWo/edit#bookmark=id.yt4ntz7whu4b)